features/resource_status.md

```diff
@@ -0,0 +1,220 @@
+
+- [ ] Disabled by default.
+
+- [ ] No impact on performance when the feature is disabled.
+
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+---
+
+## Alpha
+
+### Requirements: 
+
+**Design**
+
+- [x] RFC has been approved describing the intention of the feature as well as the user stories behind the feature. 
+
+**Config**
+
+- [x] Explicit user action is required to enable this feature (e.g. a config field, config resource, or installation action). 
+
+> Link to instructions for enabling
+    https://istio.io/latest/docs/ops/diagnostic-tools/istioctl-analyze/
+
+**Docs**
+
+- [x] Reference docs are published to preliminary.istio.io or the Istio wiki.
+- [x] Basic feature docs are published on preliminary.istio.io describing what the feature does, how to use it, and any caveats. 
+- [x] Release notes entries added as appropriate
+- [x] Upgrade notes entries added as appropriate
+
+**Tests**
+
+- [x] Automated integration tests cover core use cases with the feature enabled. 
+- [x] When disabled, the feature does not affect system stability or performance. 
+
+**API**
+
+- [x] Initial API review.
+
+**Approvals**
+
+- [x] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [x] The TOC has reviewed and approved promotion of the feature as part of the
+	roadmap for a release.
+
+**Promotion**
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+
+---
+
+## Beta
+
+### Requirements: 
+
+**Design**
+
+- [x] Design doc describing the intention of the feature, how it will be
+	implemented, and any thoughts on how to test the feature has been approved by
+	relevant work group leads
+- [x] Feature coverage and test plans written and approved.
+
+**Docs** 
+
+- [x] Documentation on istio.io includes performance expectations; may have caveats. 
+- [x] Documentation on istio.io includes samples/tutorials. 
+- [x] Documentation on istio.io includes appropriate glossary entries. 
+- [x] All new documentation containing user actions includes istio.io tests.
+- [ ] Release notes have been added. 
+- [ ] Upgrade notes have been added. 
+
+**Tests**
+
+- [x] Integration tests cover feature edge cases
+- [x] End-to-end tests cover samples/tutorials
+- [x] Fixed issues have tests to prevent regressions
+- [x] Stability/stress test suite includes coverage for the feature.
+
+    https://github.com/istio/istio/issues/30179
+
+**Performance**
+
+- [x] Feature coverage and test plans written and approved 
+- [x] Tests exist with the feature enabled that can be integrated with our automated performance testing.
```

Comment:
Can you expand on this? Where is the plan and tests?

Reply:
@howardjohn this I am not really sure, I think @therealmitchconnors probably can help more on this one.